### PR TITLE
Fixed page MarkUp & Design

### PR DESCRIPTION
**Pull request by saison**
plz Review
@shunjikonishi , @da0shi 

(This is generally the same as SinglePage)

### CODE REVIEW

File: pages.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (shunjikonishi) [inline pages.html]: It's wrong.

File: sass/_pages.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.singlePost{
+  .thumbnail{
+    img{
+      width: 100%;
+      height: auto;
+    }
+  }
+  .givery_mission{
```
Comment (shunjikonishi) [inline sass/_pages.scss]: You are using double underscore style in css definition.
So you should not use such a snake case definition.

**Comment by shunjikonishi** [comment]
I checked both of #2 and #3.
Totally, I think there are two bad practice in these codes.

## 1. Same contents in several pages.

Each pages have header, footer, and something same.  
Don't write same code in each pages.

There are many way to avoid this.  
For example, you can use gulp plugin.

## 2. Don't split the style for same class.

The style for .singlePost is in multiple files.
If you want to split css, and concatenate all, each css should have independent definitions.

I　don't think you need to split css.
Single css is enough in this case.

**Comment by saison** [comment]
1. Same contents in several pages.  => #5

**Comment by saison** [comment]
plz Re-review
@shunjikonishi

**Comment by shunjikonishi** [comment]
Same as #4.

Rebase and push again.

**Comment by saison** [comment]
this branch rebase master

plz review
@shunjikonishi

**Comment by shunjikonishi** [comment]
same as #4

**Comment by saison** [comment]
plz review
@shunjikonishi

**Comment by shunjikonishi** [comment]
LGTM

**Pull request merged**